src/backendCalls/metadata.ts

```diff
@@ -0,0 +1,34 @@
+import { sleep } from '../utils/funcUtils';
+import datasets from '../data/datasets';
+import { dataset, datasetOutline } from '../dataModels/datasets';
+
+export const getDsMetadata: (dsId: string) => dataset = (dsId) => {
+    // currently, looks up in the hardcoded datasets object;
+    // later, will get the info from a backend API
+    
+    sleep(2);
```

Comment:
Why are we sleeping before getting the metadata?

Reply:
This is just to simulate some waiting time to get the results from backend.